samples/sensor/bmp180/CMakeLists.txt

```diff
@@ -0,0 +1,8 @@
+# SPDX-License-Identifier: Apache-2.0
+
+cmake_minimum_required(VERSION 3.20.0)
+find_package(Zephyr REQUIRED HINTS $ENV{ZEPHYR_BASE})
+project(bmp180)
+
+FILE(GLOB app_sources src/*.c)
+target_sources(app PRIVATE ${app_sources})
```

Comment:
Please add the sample application in a separate commit (in the same PR)

Reply:
Should I first remove the sample, leaving driver support only with a single squashed commit, and then  create another commit with sample?

Reply:
done